docs/api.rst

```diff
@@ -20,22 +20,37 @@ API
 .. _calibration_ref:
 
 
-:mod:`tedana.model`: Modeling TE-dependence
---------------------------------------------------
+:mod:`tedana.decay`: Modeling signal decay across echoes
```

Comment:
I think we should keep the name `model`, here, as it's relatively agnostic to the kind of TE-dependence we're modelling. For example, for thermal noise we'd like to confirm that it stays stable across all TEs -- I'm not sure it makes sense to call that `decay`.

Reply:
The reason I put model below decay is that decay comes first in the workflow, so I think there's a sort of flow there, but I didn't change the names of the modules.

Reply:
Ah, ok, sorry ! Wasn't clear from the diff.